src/commandDetails/games/blackjack.ts

```diff
@@ -0,0 +1,295 @@
+// Sapphire Specific:
+// eslint-disable-next-line no-unused-vars, @typescript-eslint/no-unused-vars
+import { container } from '@sapphire/framework';
+import {
+  Colors,
+  EmbedBuilder,
+  ChatInputCommandInteraction,
+  ButtonBuilder,
+  ButtonStyle,
+  ActionRowBuilder,
+  ButtonInteraction,
+  ComponentType,
+} from 'discord.js';
+import {
+  adjustCoinBalanceByUserId,
+  getCoinBalanceByUserId,
+  UserCoinEvent,
+} from '../../components/coin';
+import { getCoinEmoji, getEmojiByName } from '../../components/emojis';
+import {
+  BlackjackAction,
+  BlackjackHand,
+  BlackjackStage,
+  CardSuit,
+  endGame,
+  GameState,
+  performGameAction,
+  startGame,
+} from '../../components/games/blackjack';
+import { pluralize } from '../../utils/pluralize';
+import {
+  CodeyCommandDetails,
+  CodeyCommandOptionType,
+  SapphireMessageExecuteType,
+  SapphireMessageResponse,
+} from '../../codeyCommand';
+
+// CodeyCoin constants
+const DEFAULT_BET = 10;
+const MIN_BET = 10;
+const MAX_BET = 1000000;
+
+// Game buttons: Hit, Stand, Quit
+const hit = new ButtonBuilder().setCustomId('hit').setLabel('Hit 🇭').setStyle(ButtonStyle.Primary);
+const stand = new ButtonBuilder()
+  .setCustomId('stand')
+  .setLabel('Stand 🇸')
+  .setStyle(ButtonStyle.Primary);
```

Comment:
I sorta "revamped" the design of the buttons so that the emojis would properly display on the button, and so the Hit button was changed to ButtonStyle.Success (i.e. green) to avoid clashing with the emoji's blue theme.

Reply:
Looks good!